### PR TITLE
Add resource attributes to the base template (for wb-main, wb-webpage, wb-publisher)

### PR DESCRIPTION
**Pull request by ctopCanada**
WET - 123 Add resource attributes to the base template (for wb-main, wb-webpage, wb-publisher)

### CODE REVIEW

File: site/pages/gcweb-theme/static-header-footer/bootstrap-3.html
Comment (GormFrank) [inline site/pages/gcweb-theme/static-header-footer/bootstrap-3.html:31]: Remove empty line here.

**Review by GormFrank — CHANGES_REQUESTED** [review]
Also remove the changes on the release notes files (files that contain the word "release" in their name).

File: site/pages/gcweb-theme/static-header-footer/bootstrap-3.html
Comment (duboisp) [inline site/pages/gcweb-theme/static-header-footer/bootstrap-3.html:15]: Missing a resource 
* at line 31 for the publisher info 
* at line 88 for the main element.

File: site/pages/gcweb-theme/static-header-footer/bootstrap-4.html
Comment (duboisp) [inline site/pages/gcweb-theme/static-header-footer/bootstrap-4.html:22]: Can you add the RDFa markup, like we have for the bootstrap 3 template, for the publisher area.
Here is line 35 to 40

File: site/layouts/splashpage.hbs
Comment (duboisp) [inline site/layouts/splashpage.hbs:23]: Can you add the same RDFa attribute as the other page on the main element here at line 38

File: site/layouts/servermessage.hbs
Comment (duboisp) [inline site/layouts/servermessage.hbs:15]: Missing RDFa resource in the main element at line 25

**Review by duboisp — CHANGES_REQUESTED** [review]

**Review by duboisp — APPROVED** [review]

**Comment by duboisp** [comment]
@ctopCanada please fix the linting error

File: site/layouts/splashpage.hbs
Comment (GormFrank) [inline site/layouts/splashpage.hbs:38]: Remove container class on this one:
```suggestion
		<main role="main" property="mainContentOfPage" resource="#wb-main" typeof="WebPageElement">
```

**Review by GormFrank — CHANGES_REQUESTED** [review]

**Review by GormFrank — APPROVED** [review]

**Review by duboisp — DISMISSED** [review]
@ctopCanada can you squash all the commits into 1 commits.

This was pre-approved, so after your squash we will be able to merged

**Pull request merged**
